app/javascript/components/motif-form.js

```diff
@@ -43,10 +48,14 @@ class MotifForm {
     document.querySelectorAll(noSecretariatInputs).forEach(input =>
       input.addEventListener('change', e => this.toggleSecretariat())
     )
-    this.reservableOnlineCheckbox.addEventListener('change', e => this.toggleSectorisation())
+    this.reservableOnlineCheckbox.addEventListener('change', e => {
+      this.toggleSectorisation()
+      this.toggleRdvsEditableAndCancellable()
```

Comment:
Un oubli je crois :wink: :

```suggestion
      this.toggleRdvsEditable()
```

Reply:
Oui bien vu merci :) 🙌 !